src/libPMacc/include/memory/buffers/GridBuffer.hpp

```diff
@@ -203,9 +203,11 @@ class GridBuffer
      *        if dataPlace=BORDER than copy other GUARD to my BORDER
      * @param receive a Mask which describes the directions for the exchange
      * @param guardingCells number of guarding cells in each dimension
-     * @param sizeOnDevice if true, internal buffers have their size information on the device, too
+     * @param communicationTag unique tag/id for communication
+     * @param sizeOnDeviceSend if true, internal send buffers have their size information on the device, too
```

Comment:
as this interface shall explain a user if he/she should choose to create this attribute on the device or not, let's write

```
if true, internal send buffers must store their size additionally on the device
(as we keep this information coherent with the host, it might impact performance)
```

similar in the line below